frontends/qt/setup.nsi

```diff
@@ -103,6 +104,12 @@ Section -Main SEC0000
     #File /r /x Makefile* @abs_top_srcdir@/doc\*.*
     SetOutPath $INSTDIR
     WriteRegStr HKCU "${REGKEY}\Components" Main 1
+
+    # Explicitly copy the icon file to the installation directory
+    File "${ICONDIR}\icon.ico"
+
+    # Create a shortcut on the desktop
+    CreateShortCut "$DESKTOP\${APP_NAME}.lnk" "$INSTDIR\${APP_EXE}" "" "$INSTDIR\icon.ico" 0
```

Comment:
the app already embeds an icon doesn't it? can you try simply setting the icon string to `""` and not copying the icon again to the install folder?

Reply:
I meant to try like this:

    CreateShortCut "$DESKTOP\${APP_NAME}.lnk" "$INSTDIR\${APP_EXE}" "" "" 0

Reply:
ping @jadzeidan

Reply:
Tested and that works 👍 

Will update PR